src/lib/app/generate-code.ts

```diff
@@ -671,3 +603,128 @@ export const getViewDbTypeDeclaration = (
   )}>`;
   return [comment, declaration].join('\n');
 };
+
+export const writeFrieda = async (
```

Comment:
I don't like that we have two files `frieda.js` and `frieda.ts` in the javascript case. We should change the latter to `frieda.d.ts`, which, to me indicates it's purpose better. 

Not sure if that'l work, but...